terraform/modules/forest_node/user-data.tpl

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+NEW_USER="${NEW_USER}"
+VOLUME_NAME="${VOLUME_NAME}"
+
+# Create a login user
+useradd -m "${NEW_USER}"
+
+# Create .ssh directory for created user
+mkdir --parents -- "/home/${NEW_USER}/.ssh"
+chown "${NEW_USER}:${NEW_USER}" "/home/${NEW_USER}/.ssh"
+chmod 0700 "/home/${NEW_USER}/.ssh"
+
+# Check if authorized_keys file exists
+if [ ! -f "/root/.ssh/authorized_keys" ]; then
+  echo "No authorized_keys file found. Skipping copying..."
+else
+  # Copy public key from existing root to new user
+  cp /root/.ssh/authorized_keys "/home/${NEW_USER}/.ssh/authorized_keys"
+  chown "${NEW_USER}:${NEW_USER}" "/home/${NEW_USER}/.ssh/authorized_keys"
+  chmod 0600 "/home/${NEW_USER}/.ssh/authorized_keys"
+fi
```

Comment:
```suggestion
if [ -f "/root/.ssh/authorized_keys" ]; 
  : Copy public key from existing root to new user
  cp /root/.ssh/authorized_keys "/home/${NEW_USER}/.ssh/authorized_keys"
  chown "${NEW_USER}:${NEW_USER}" "/home/${NEW_USER}/.ssh/authorized_keys"
  chmod 0600 "/home/${NEW_USER}/.ssh/authorized_keys"
fi
```

Reply:
this is a good thought but i think you're missing `then` i.e `if [ -f "/root/.ssh/authorized_keys" ]; then`